build/target-repository/.github/workflows/publish_docker_images.yaml

```diff
@@ -29,6 +29,14 @@ jobs:
                   username: ${{ secrets.DOCKER_USERNAME }}
                   password: ${{ secrets.DOCKER_PASSWORD }}
 
+            - name: Update DockerHub overview
+              uses: peter-evans/dockerhub-description@v2
+              with:
+                  username: ${{ secrets.DOCKER_USERNAME }}
+                  password: ${{ secrets.DOCKER_PASSWORD }}
+                  repository: rector/rector
+                  readme-filepath: ./docs/how_to_run_rector_in_docker.md
+
```

Comment:
Thanks :+1:

Reply:
It seems need token? https://github.com/rectorphp/rector/runs/2865521242#step:5:1

```
with:
   username: ***
    password: ***
    repository: rector/rector
    readme-filepath: ./docs/how_to_run_rector_in_docker.md
  env:
    COMPOSER_ROOT_VERSION: dev-main
Acquiring token
Sending PATCH request

Error: FORBIDDEN
```